routes/jobs.js

```diff
@@ -3,18 +3,22 @@ const router = new express.Router({ mergeParams: true });
 const Jobs = require('../models/jobs-model');
 
 router.get('/new', (req, res) => {
-  res.render('jobs-new', { id: req.params.id });
+  res.render('jobs-new', { accountId: req.params.accountId });
 });
 
 router.post('/new', (req, res, next) => {
   req.checkBody('title', 'Job title is required').notEmpty();
 
   if (req.validationErrors()) {
-    const body = Object.assign({ errors: req.validationErrors() }, { id: req.params.id }, req.body);
+    const body = Object.assign(
+      { errors: req.validationErrors() },
+      { accountId: req.params.accountId },
+      req.body
+    );
     return res.render('jobs-new', body);
   }
-  return new Jobs(Object.assign(req.body, { accountId: req.params.id })).save()
-    .then(() => res.redirect(`/${req.params.id}`))
+  return new Jobs(Object.assign(req.body, { accountId: req.params.accountId })).save()
+    .then(() => res.redirect(`/${req.params.accountId}`))
```

Comment:
same as above => to avoid duplication `const accountId = req.params.accountId` ?